applications/FluidDynamicsApplication/custom_elements/fluid_element.h

```diff
@@ -433,7 +433,7 @@ class FluidElement : public Element
         TElementData& rData,
         MatrixType& rMassMatrix);
 
-    virtual void AddBoundaryIntegral(
+    virtual void AddBoundaryTraction(
```

Comment:
Could you add a doxygen comment explaining what this function should do in derived implementations?

Reply:
Sure! Actually, I completely forgot to do it...

Reply:
Done, if it isn't clear enough tell me something.